### ISSUE TITLE
File search

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#29]

**Comment by bmix** [comment]
Should be configurable, so the OS' default way to search for files could be used or an external application (Everything on Windows is pretty nice, locatedb on Unixoid systems, Spotlight on macOS, etc.)

**Comment by ThePlagueD0ctor** [comment]
Also i'd like to have "search by text" feature similar to ctrl+F7 on TC. It's so important feature for devs

**Comment by mherrmann** [comment]
@ThePlagueD0ctor I agree that would be important.

**Comment by ThePlagueD0ctor** [comment]
Yes, thanks! Also i want to clarify that i meant searching text in files. For example i have a folder with stored procedures and i want to find those that contains some phrase (for example i need all procedures that call some function). That is extremely important thing).
Thank you!

**Comment by mherrmann** [comment]
Yes, that's what I thought you meant :-)

**Comment by tmssngr** [comment]
It should support intuitive search by file name, file type (file and/or directory), file content.

**Comment by eandmsz** [comment]
It should also support the same advanced features as Total Commander:
Find files which have:
 Date between...
 Not older than...
 Older than...
 File size: greater, less, equal to the specified, bytes, kB, MB, GB
 Find duplicate files based on: same name, same size, same contents

**Comment by Chaiavi** [comment]
What is this request ?

Search for a file (by regex/time/date etc)?
Or search text IN a file ?

Or both ?

**Comment by mherrmann** [comment]
I suppose the former first, then the second.

**Comment by Chaiavi** [comment]
Agreed, both are very important in the order you mentioned.

**Comment by nojaf** [comment]
Regex search on files names would be very nice.

**Comment by lawrence-laz** [comment]
What's the status of this?
